packages/remix/src/build.ts

```diff
@@ -167,17 +168,13 @@ module.exports = config;`;
         });
       }
     }
-    remixConfig = await chdirAndReadConfig(entrypointFsDirname);
```

Comment:
This second read `readConfig()` call was not actually doing anything, since `readConfig()` uses `await import()` and thus the module was cached and didn't pick up our patches.